src/shared/api/baseApi.ts

```
@@ -1,9 +1,45 @@
+      timeout: 10000,
+    });
+  }
+
+  protected async get<T = any>(
+    url: string,
+    config?: AxiosRequestConfig
+  ): Promise<AxiosResponse<T>> {
+    return this.axiosInstance.get<T>(url, config);
+  }
+
+  protected async post<T = any>(
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)